src/api/app/controllers/webui/status_messages_controller.rb

```diff
@@ -35,7 +35,7 @@ def acknowledge
     status_message = StatusMessage.find(params[:id])
     if status_message.acknowledge!
       RabbitmqBus.send_to_bus('metrics', "user.acknowledged_status_message status_message_id=#{status_message.id}")
-    else
+    elsif !status_message.acknowledged?
       flash.now[:error] = "Could not accept status message: #{status_message.errors.full_messages.to_sentence}"
```

Comment:
The flash won't have any error message when `status_message.acknowledge!` is `false` due to a user already acknowledged. We could add an error to `status_message` so this would be addressed.

Reply:
Oh wait, this won't happen because we're checking again if the status message is acknowledged.